__tests__/endpoint.tests/endpoint.test.js

```diff
@@ -156,6 +156,28 @@ describe("articles", () => {
           expect(body.msg).toBe("Please enter a valid query");
         });
     });
+
+    test("GET:200 returns an array of articles with topic being the entered value in topic query", () => {
+      return request(app)
+        .get("/api/articles?topic=mitch")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body.articles)).toBe(true);
+          expect(body.articles.length).toBeGreaterThan(0);
+          body.articles.forEach((article) => {
+            expect(article.topic).toBe("mitch");
+          });
+        });
+    });
+
+    test("GET:404 returns error when topic does not exist", () => {
+      return request(app)
+        .get("/api/articles?topic=not-a-topic")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Topic not found");
+        });
+    });
```

Comment:
What happens if you search for a topic that exists but has no articles?

Reply:
@nc-mark-t Since topic doesn't have its own table and is taken directly from an article table, this would never be the case

Reply:
topics does have it's own table and `paper` is an example of this exact thing